cnf-certification-test/lifecycle/suite.go

```diff
@@ -183,6 +189,22 @@ func testContainersPreStop(env *provider.TestEnvironment) {
 	}
 }
 
+func testContainersPostStart(env *provider.TestEnvironment) {
+	badContainers := []string{}
+	for _, cut := range env.Containers {
+		logrus.Debugln("check container ", cut.String(), " post start lifecycle ")
+
+		if cut.Lifecycle == nil || (cut.Lifecycle != nil && cut.Lifecycle.PostStart == nil) {
+			badContainers = append(badContainers, cut.Name)
+			tnf.ClaimFilePrintf("%s does not have postStart defined", cut)
+		}
+	}
+	if len(badContainers) > 0 {
+		tnf.ClaimFilePrintf("Containers have been found missing lifecycle postStart definitions: %v", badContainers)
+		ginkgo.Fail("Containers have been found missing lifecycle postStart definitions.")
```

Comment:
For this TC, and the `preStop` TC, should we switch them to use the `testhelper.AddTestResultLog` helper func?

Reply:
fixed